src/runtime/rpc/rpc_module.cc

```diff
@@ -300,6 +305,22 @@ std::shared_ptr<RPCSession> RPCModuleGetSession(Module mod) {
   return rmod->sess();
 }
 
+inline void CacheFlush(const char* p, unsigned int allocation_size) {
```

Comment:
document the function, describe the parameters and the motivation behind the cache flush operation